src/Symfony/Component/Serializer/Tests/SerializerTest.php

```diff
@@ -716,6 +717,23 @@ public function testDeserializeInconsistentScalarType()
         $serializer->deserialize('"42"', 'int', 'json');
     }
 
+    /**
+     * @requires PHP 8.1
+     */
+    public function testDeserializeWhenObjectNotInstantiable()
```

Comment:
What about moving that test to the `AbstractNormalizerTest` as it's only related to the denormalization?

Reply:
I've moved it 👍